### PR TITLE
update terraform-docs location

### PR DESCRIPTION
**Pull request by bacongobbler**
cc @khos2ow

### CODE REVIEW

File: Food/terraform-docs.lua [hunk 1]
```diff
@@ -4,18 +4,17 @@ local version = "0.11.2"
...
             arch = "amd64",
-            url = "https://github.com/segmentio/" .. name .. "/releases/download/v" .. version .. "/" .. name .. "-v" .. version .. "-darwin-amd64",
-            -- shasum of the release archive
-            sha256 = "554512aabf8f4cb8579f797dea21085711a21eefa1492456c4854991326cebb9",
+            url = "https://github.com/terraform-docs/" .. name .. "/releases/download/v" .. version .. "/" .. name .. "-v" .. version .. "-darwin-amd64.tar.gz",
+            sha256 = "ac7ce685dea91212199c58de2379b419a5f512f70eda08b435176c1b5594716c",
             resources = {
                 {
-                    path = name .. "-v" .. version .. "-darwin-amd64",
+                    path = name,
```
Comment (khos2ow) [inline Food/terraform-docs.lua:17]: I'm not aware of inner-mechanic of how gofish works, but I wanted to make sure `path` being renamed from `terraform-docs-vX.Y.Z-OS-ARCH` to `terraform-docs` doesn't break usage for end users.

**Review by khos2ow — APPROVED** [review]
Thanks, LGTM!

File: Food/terraform-docs.lua (hunk 1 above)
Comment (bacongobbler) [reply Food/terraform-docs.lua:17]: yes.

Comment (khos2ow) [reply Food/terraform-docs.lua:17]: Awesome!

**Pull request merged**